ui/src/admin/components/chronograf/NewUserTableRow.js

```diff
@@ -0,0 +1,217 @@
+import React, {Component, PropTypes} from 'react'
+
+import Authorized, {SUPERADMIN_ROLE} from 'src/auth/Authorized'
+
+import Dropdown from 'shared/components/Dropdown'
+import SlideToggle from 'shared/components/SlideToggle'
+
+import {USERS_TABLE} from 'src/admin/constants/chronografTableSizing'
+import {DEFAULT_ORG_ID} from 'src/admin/constants/dummyUsers'
+
+class NewUserTableRow extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      userName: '',
```

Comment:
seems like we could get rid of `user` in each of these since this component contemplates a UserTableRow and the only state is about the user

Reply:
Done